modules/RTC/RTCUtils.js

```diff
@@ -353,6 +116,7 @@ function getConstraints(um, options = {}) {
  * @param {Object} options.frameRate - used only for dekstop sharing.
  * @param {Object} options.frameRate.min - Minimum fps
  * @param {Object} options.frameRate.max - Maximum fps
+ * @param {string} options.resolution - resolution constraints.
```

Comment:
Why is this here now? `resolution` was part of the old flow, the new flow uses `constraints`, which is already a parameter, with the video constrainsts.

Reply:
Maybe I am confused, I was told that some client apps still use options.resolution to override the default constraints. If that is not the case anymore, I will remove resolution from the options.

Reply:
Right now, the resolution option in config.js is only used in Firefox, well and anything not using the new gUM flow. I always disliked that the new `constraints` option ignored the legacy `resolution` option, so there is that.

I think what we need here is a decission, more than anything.

Reply:
Ok, I will bring it up with tin the standup to see what we want to do here.